### PR TITLE
[SWE-Paddle] Add task PaddlePaddle__Paddle_74491

### PR DESCRIPTION
**Pull request by Echo-Nie**
# 关联

- SWE-Paddle 共建 issue: https://github.com/PaddlePaddle/Paddle/issues/79447
- 来源 PR: https://github.com/PaddlePaddle/Paddle/pull/74491

## 说明

- 新增 SWE-Paddle 任务 `PaddlePaddle__Paddle-74491`

@sunzhongkai588

## 验证结果

| 状态 | Existing `dim` P2P | `requires_grad` state F2P | Invalid assignment F2P |
| --- | --- | --- | --- |
| Base + `tests/test.patch` | PASS | FAIL，符合预期 | FAIL，符合预期 |
| Base + test patch + `solution/code.patch` | PASS | PASS | PASS |

Base：

```text
P2P: 1 passed
F2P: 2 failed, 1 passed
tests/test.sh exit code: 1
```

Solution：

```text
P2P: PASS
F2P: PASS
Full target test: 3 passed in 0.08s
tests/test.sh: 3 passed in 0.08s
```

### CODE REVIEW

File: swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# 为 Tensor 增加 `requires_grad` 属性
+
+## 详细描述
+
+Paddle 目前通过 `stop_gradient` 控制 Tensor 是否参与梯度计算，但还不支持 `requires_grad` 属性。
+
+需要为动态图 Tensor、静态图 Variable 和 PIR Value 增加 `requires_grad`。该属性与 `stop_gradient` 的含义相反：
```
Comment (sunzhongkai588) [inline swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md]: PIR Value 是啥？

File: swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# 为 Tensor 增加 `requires_grad` 属性
+
+## 详细描述
+
+Paddle 目前通过 `stop_gradient` 控制 Tensor 是否参与梯度计算，但还不支持 `requires_grad` 属性。
```
Comment (sunzhongkai588) [inline swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md]: > 但还不支持 `requires_grad` 属性。

感觉这句话有点怪，要不加一句为了增加兼容性之类的？

File: swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md (hunk 1 above)
Comment (Echo-Nie) [reply swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md]: 一个类型？相当于就是PIR 的一个Tensor-like 的符号对象

image

File: swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md (hunk 2 above)
Comment (Echo-Nie) [reply swe-paddle/tasks/PaddlePaddle__Paddle-74491/instruction.md]: Done

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
